### DISCUSSION TITLE
Explore: Log browser filename picker is not expandable

### DISCUSSION THREAD

**Discussion by dskrypa** [grafana/grafana#46518] [Observability]
**What happened**:
In the Explore log browser, long file names from Loki are hard to read / require a lot of horizontal scrolling:
![image](https://user-images.githubusercontent.com/7505342/158066933-d8d18f58-a34d-4767-88c3-bf4d49b2ba9c.png)

**What you expected to happen**:
It would be easier to select a file name if the div was resizable.  Setting `resize: both` via devtools is the workaround I have used:
![image](https://user-images.githubusercontent.com/7505342/158067136-a06b4bfb-baaf-4101-a002-1f10ee5a0e8d.png)

This happens regardless of whether the `job` label is selected in part 1.

**How to reproduce it (as minimally and precisely as possible)**:
- Add static scrape configs for Promtail that read one or more logs with long paths
- Open Explore and expand `Log browser`
- Try to resize the filename list

**Anything else we need to know?**:
I'm relatively new to Grafana, but it appears that this picker is very similar to the label value pickers used in the Metrics browser for data from Prometheus.  I assume I would see more selection panels if I had more labels configured.  I think having any selection panel be resizable if its contents exceed the width of the panel would be helpful / useful.

**Environment**:
- Grafana version: 8.3.6, 8.4.3
- Data source type & version: Loki / Promtail 2.4.2
- OS Grafana is installed on: Both FreeBSD 12.2 and Windows 10 (via WSL with Ubuntu)
- User OS & Browser: Windows 10, Firefox, Edge, Chrome
- Grafana plugins: none
- Others: none

**Comment by ivanahuckova** [comment]
Hello @dskrypa, I like this idea a lot! 🤩 But because this is a feature request, I am going to transfer it to discussion as we store all feature requests as discussions before we commit to working on them. I'll bring this up also with our UX designers.

**Reply by dskrypa** [reply]
Understood, thanks :)

**Comment by ivanahuckova** [comment]
@mary-sitzenstatter @niathadgu what do you think about making rows in label browser resizable. This way, when you have long label values, you can resize the ones you want/need. Something like this: 

https://user-images.githubusercontent.com/30407135/158147247-8fe73cb8-1244-47f4-912c-180875ae1ca5.mov

**Reply by Alan01252** [reply]
Like this a lot too!

**Comment by mary-sitzenstatter** [comment]
@ivanahuckova @dskrypa - I think this looks great and makes SO MUCH sense given how long some of these label / value names can be. I know that @diegoadams is working on the Loki query editor and wonder if he has any additional feedback? But, again, in my view I think this seems incredibly useful. 

cc: @niathadgu
